compiler-cli/Cargo.toml

```diff
@@ -78,6 +78,8 @@ lsp-types = "0.92"
 fslock = "0.2.1"
 # Compact and cheap to clone immutable string type
 smol_str = "0.1"
+# Open generated docs in browser
+opener = "0.6"
```

Comment:
We're pulling in the [`opener`](https://crates.io/crates/opener) crate to handle opening the browser for us.

This is also what `cargo doc --open` uses when there is no browser set in the workspace config: https://github.com/rust-lang/cargo/blob/c9b89c67723b93151f023ac3a932a471fe9fa6a0/src/cargo/ops/cargo_doc.rs#L66-L71